### PR TITLE
LPD-24103 Restore the test to the first code because the modification caused it to fail

### PR DESCRIPTION
**Pull request by vngoliveira**
[LPD-24103](https://liferay.atlassian.net/browse/LPD-24103)

### CODE REVIEW

**Comment by marcelabc** [comment]
Just started reviewing :)

File: modules/apps/object/object-web/src/test/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemObjectProviderTest.java [hunk 1]
```diff
@@ -188,19 +188,18 @@ public void testGetInfoItemProxyObjectEntryNoSuchInfoItemException()
 	public void testGetInfoItemProxyObjectEntryNullHttpServletRequest()
 		throws Exception {
 
-		ObjectEntry objectEntry = Mockito.mock(ObjectEntry.class);
-
 		String externalReferenceCode = RandomTestUtil.randomString();
+		ObjectEntry objectEntry = Mockito.mock(ObjectEntry.class);
```
Comment (marcelabc) [inline modules/apps/object/object-web/src/test/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemObjectProviderTest.java:192]: You don't need to change the order of creation of these variables (externalReferenceCode, objectEntry). The creation does not impact the assert method.

Can you keep it the way it was, please?

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by vngoliveira** [comment]
ci:test:sf

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**